File: 프로그래머스/lv0/120956. 옹알이 （1）/옹알이 （1）.py
def solution(babbling):
    count = 0
    check = ["aya", "ye", "woo", "ma"]
    for b in babbling:
        for c in check:
            if c in b:
                b = b.replace(c," ",1)
        if not b.split():
            count += 1
        print(b.split())
    return count

File: 프로그래머스/0/120956. 옹알이 （1）/옹알이 （1）.cpp
#include <string>
#include <vector>

using namespace std;

int solution(vector<string> babbling) {
    int answer = 0;
    int idx;
    vector<string> candi = {"aya", "ye", "woo", "ma"};
    for (string s: babbling) {
        for (string c: candi) {
            int len = c.size();
            if ((idx = s.find(c, 0)) != string::npos) {
                s.replace(idx, len, "_");
            }
        }
        while ((idx = s.find("_", 0)) != string::npos) {
            s.replace(idx, 1, "");
        }
        if (s == "") answer++;
    }
    return answer;
}
